dallinger/frontend/static/scripts/tracking/scribe-dallinger.js

```diff
@@ -0,0 +1,42 @@
+var ScribeDallingerTracker = function(config) {
+  if (!(this instanceof ScribeDallingerTracker)) return new ScribeDallingerTracker(config);
+
+  this.config = config;
+};
+
+ScribeDallingerTracker.prototype.tracker = function(info) {
+  var config = this.config;
+  var path = info.path;
+  var value = info.value || {};
+  var data = new FormData(); 
+  if (config.base_url) {
+    // Only track events
+    if (path.indexOf('/events/') < 0) {
+      return;
+    }
+
+    // Remove possible PII
+    if (value.fingerprint) {
+      delete value.fingerprint;
+    }
+    if (value.visitorId) {
+      delete value.visitorId;
+    }
+
+    data.append('Event.1.EventType', 'TrackingEvent');
+    if (config.participant_id) data.append('participant_id', config.participant_id);
+    if (config.assignment_id) data.append('Event.1.AssignmentId', config.assignment_id);
+    data.append('details', JSON.stringify(value));
+
+    var xhr = new XMLHttpRequest();
```

Comment:
This is fine for the proof of concept, but maybe since we have websocket support we should use it?

Reply:
Hm, I don't know if we want to setup the websocket stuff when working on external sites. Is there any advantage to using a websocket versus using the notification route that uses a redis worker to store the events?

Reply:
Just that it would probably have less overhead per event. Let's not worry about it for now.